flutter_frontend_server/lib/server.dart

```diff
@@ -169,3 +182,44 @@ Future<int> starter(
   frontend.listenAndCompile(compiler, input ?? stdin, options, completer);
   return completer.future;
 }
+
+class _ToStringVisitor extends RecursiveVisitor<void> {
```

Comment:
I  would make this public (because we don't publish this package, so who cares) and then unit test as normal

Reply:
This now has tests.  The tests .. work because .. uh .. we do weird magic when you build the server.  And somehow package:test becomes available.  Should I declare it explicitly as a dev dependency or not though?